openff/toolkit/typing/engines/smirnoff/parameters.py

```diff
@@ -3706,6 +3712,15 @@ def create_force(self, system, topology, **kwargs):
             atom_matches, topology, [(atom,) for atom in topology.atoms]
         )
 
+    def _apply_switching_function(self, force: "openmm.NonbondedForce"):
```

Comment:
I'm not smart enough to really follow the type annotation here, or how it related to the `if TYPE_CHECKING: import openmm` line above, so maybe we could remove this. It's not a techncal issue, I'm just not smart enough to follow, and that means that average contributors may not be smart enough either.